src/LearningObjectDownload/Interactor.ts

```diff
@@ -233,7 +237,8 @@ async function bundleLearningObject({
       }),
       bundleChildren({
         children: learningObject.children,
-        prefix
+        prefix,
+        requester
```

Comment:
This will help minimize git diff:
```suggestion
        requester,
```